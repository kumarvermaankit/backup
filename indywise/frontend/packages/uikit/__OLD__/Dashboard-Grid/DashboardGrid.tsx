import * as React from 'react';
import styled from 'styled-components';

interface FooProp {
  left: any;
  right: any;
  style_left: any;
  style_right: any;
}

const DashboardGrid = (props: FooProp) => {
  return (
    <div>
      <Wrapper>
        <FlexLeft style={props.style_left}>{props.left}</FlexLeft>
        <FlexRight style={props.style_right}>{props.right}</FlexRight>
      </Wrapper>
    </div>
  );
};

export default DashboardGrid;

const Wrapper = styled.div`
  display: flex;
  width: 100%;
  flex-wrap: wrap;
  margin: 0;
  padding: 0;
  list-style: none;
  overflow-y: hidden;
`;

const FlexLeft = styled.div`
  flex: 1;
  height: 100vh;
  overflow-y: hidden;
`;

const FlexRight = styled.div`
  flex: 4;
  height: 100vh;
  background: #ffffff;
  box-shadow: 0px 6px 36px rgba(164, 164, 164, 0.35);
  border-top-left-radius: 30px;
  padding-left: 20px;
  padding-right: 20px;
  overflow-y: scroll;
`;
