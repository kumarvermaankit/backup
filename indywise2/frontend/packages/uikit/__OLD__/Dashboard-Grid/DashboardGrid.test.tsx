import * as React from 'react';
import { create } from 'react-test-renderer';

import DashboardGrid from './DashboardGrid';

describe('Dashboard Grid', () => {
  it('should render correctly', () => {
    const component = create(
      <DashboardGrid
        left="Left Children"
        right="Right Children"
        style_left={{}}
        style_right={{}}
      />
    ).toJSON();

    expect(component).toMatchSnapshot();
  });
});
