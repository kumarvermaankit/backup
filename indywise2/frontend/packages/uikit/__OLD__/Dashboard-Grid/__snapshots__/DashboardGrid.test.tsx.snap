// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Grid should render correctly 1`] = `
<div>
  <div
    className=""
  >
    <div
      className=""
      style={Object {}}
    >
      Left Children
    </div>
    <div
      className=""
      style={Object {}}
    >
      Right Children
    </div>
  </div>
</div>
`;
